src/controllers/article.js

```diff
@@ -0,0 +1,35 @@
+import { validateArticle, slug } from '../utils/article';
+import { Article } from '../models';
+
+/**
+ * @name AddArticles
+ * @description This is the method for inserting articles
+ * @param {object} req The request object
+ * @param {object} res The response object
+ * @returns {int} Returns the inserted article after success
+ */
+export const AddArticles = async (req, res) => {
+  const validate = await validateArticle(req.body);
+  if (validate.fails()) return res.status(400).json(validate.errors.all());
+  req.body.slug = slug(req.body.title);
```

Comment:
Let us not mutate.

Reply:
@Temmyogunbo do I create another object, with the updated article?